### DISCUSSION TITLE
A question about v-coco dataset

### DISCUSSION THREAD

**Discussion by ltttpku** [fredzzhang/upt#67] [Q&A] [answered]
Hi, thank you for your code; But i have a question about the V-COCO dataset you implemented.

V-COCO is a subset of COCO dataset and has 10, 396 images (`5,400 for training` and `4,964 for testing`) as stated in paper.
However, when executing your code for v-coco dataset, I found` len(trainset)==4969 `and `len(testset)==4532`. The reported number of training images does not match the actual one.

**Comment by fredzzhang** [comment, marked as answer]
Hi @ltttpku,

If I remember this correctly, there are images without sufficient bounding box annotations. So I removed those images. It's the same situation with HICO-DET as well. The authors claimed there are `38118` training images, while in fact only `37633` of those images have bounding box annotations.

Fred.

**Comment by ltttpku** [comment]
ohh, I see. Thanks!

**Reply by hutuo1213** [reply]
Hi @fredzzhang,

I found 5400-431=4969 valid actions in instances_vcoco_trainval.json. This is different from the 4971 published by PPDM, what causes this discrepancy? How are your annotation files collected? Hope to get your guidance!

yaoyaosanqi.

**Reply by fredzzhang** [reply]
Hi @yaoyaosanqi,

The files `instances_vcoco_*.json` were converted from the original annotation files released by Gupta et al. I'm not sure what the authors of PPDM did with their annotations, so I do not know why there is a discrepancy. It's possible that they didn't remove images for the verb *point*, which do not contain valid HOI triplets.

Fred.

**Reply by hutuo1213** [reply]
We looked at and ran the code, as you said. 
Cheers!
